packages/fether-electron/src/main/app/menu/template/index.js

```diff
@@ -48,40 +97,92 @@ export const template = [
         }
       }
     ]
-  }
-];
+  };
 
-if (process.platform === 'darwin') {
-  template.unshift({
-    label: app.getName(),
-    submenu: [
-      { role: 'about' },
-      { type: 'separator' },
-      { role: 'services', submenu: [] },
+  let template = [
+    fileTab,
+    editTab,
+    process.platform === 'win32' ? viewTabWindowsOS : viewTab,
+    windowTab,
+    helpTab
+  ];
+
+  if (process.platform === 'darwin') {
+    // Edit menu
+    template[1].submenu.push(
       { type: 'separator' },
-      { role: 'hide' },
-      { role: 'hideothers' },
-      { role: 'unhide' },
+      {
+        label: 'Speech',
+        submenu: [{ role: 'startspeaking' }, { role: 'stopspeaking' }]
+      }
+    );
+  }
+
+  if (process.platform === 'darwin') {
+    // Window menu
+    template[3].submenu = [
+      { role: 'close' },
+      { role: 'minimize' },
+      { role: 'zoom' },
       { type: 'separator' },
-      { role: 'quit' }
-    ]
-  });
-
-  // Edit menu
-  template[1].submenu.push(
-    { type: 'separator' },
-    {
-      label: 'Speech',
-      submenu: [{ role: 'startspeaking' }, { role: 'stopspeaking' }]
-    }
-  );
-
-  // Window menu
-  template[3].submenu = [
-    { role: 'close' },
-    { role: 'minimize' },
-    { role: 'zoom' },
-    { type: 'separator' },
-    { role: 'front' }
-  ];
-}
+      { role: 'front' }
+    ];
+  }
+
+  if (fetherApp.options.withTaskbar) {
+    // Remove Window menu tab when running as taskbar app
+    template.splice(3, 1);
+  }
+
+  return template;
+};
+
+const getContextTrayMenuTemplate = fetherApp => {
+  if (fetherApp.options.withTaskbar) {
+    const template = [
+      {
+        label: 'Show/Hide Fether',
+        click () {
+          fetherApp.win.isVisible()
+            ? fetherApp.win.hide()
+            : fetherApp.win.show();
+        }
+      },
+      { label: 'Quit', role: 'quit' }
+    ];
+
+    return template;
+  }
+};
+
+const getContextWindowMenuTemplate = fetherApp => {
+  let template = getMenubarMenuTemplate(fetherApp);
+
+  if (fetherApp.options.withTaskbar) {
+    // Remove File and Help menus in taskbar mode for window context menu
+    template.shift();
+    template.pop();
+    template.push();
+    template.push({
+      role: 'help',
+      submenu: [
+        { role: 'about' },
```

Comment:
Clicking on "about" doesn't do anything. I'd suggest to keep only one of "about" or "Learn More" and point it to https://github.com/paritytech/fether instead (as Parity.io does).

Reply:
I left both because I thought we should give the user the ability to view their current Fether version, and also finding out more about Parity and Fether:
* About - opens the window that shows the version information
* Learn More - opens https://www.parity.io/

Reply:
Oh, I see what you mean, it doesn't work because the 'about' role is only supported on macOS according to the API docs https://electronjs.org/docs/api/menu-item#roles.
I'll push an update so it only appears on macOS